src/test/app/utils/stringUtils.ts

```diff
@@ -0,0 +1,28 @@
+/* tslint:disable:no-unused-expression */
+import { expect } from 'chai'
+
+import StringUtils from 'utils/stringUtils'
+
+describe('StringUtils', () => {
+  describe('trimToUndefined', () => {
+    it('should return undefined if value is undefined', () => {
```

Comment:
what about `null`?

Reply:
Decided to code against undefined only because we tend to avoid nulls. Can add this on assumption that is candidate library for sharing with others.

Reply:
no nulls